src/lib/Components/Bidding/Components/__tests__/MarkdownRenderer-tests.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react"
+import "react-native"
+import * as renderer from "react-test-renderer"
+import { MarkdownRenderer } from "../MarkdownRenderer"
+
+describe("Markdown parser", () => {
+  it("parses", () => {
+    const md = "text with some [link](http://example.com) in it"
+    const bg = renderer.create(<MarkdownRenderer md={md} />).toJSON()
+    expect(bg).toMatchSnapshot()
+  })
```

Comment:
I wonder if there could be a better test for this, maybe a check that `example.com` isn't included in the text because that would indicate that the MD was parsed?

Reply:
👍👍

Reply:
you could also use `enzyme` to actually look check for the link and its props. 
In a test:

```javascript
   const wrapper = mount(<MarkdownRenderer md={md} />)
   expect(wrapper.find(TextLink)).toHaveLength(1)
   ...
```

Reply:
Thanks @l2succes. much better now!